src/pykeen/nn/message_passing.py

```diff
@@ -211,7 +211,7 @@ def _get_weight(self, relation_id: int) -> torch.FloatTensor:
         :return:
             A 2-D matrix.
         """
-        return self.relation_representations(indices=relation_id).squeeze(dim=0)
+        return self.relation_representations(indices=torch.as_tensor(relation_id)).squeeze(dim=0)
```

Comment:
How is this relevant? Should we be sending this to a device as well?

Reply:
`self.relation_representations` is a `RepresentationModule` itself, and thus tries to send the passed indices to the device - this does not work with Python `int`s.